src/app/shared/detail-data/detail-data.component.ts

```diff
@@ -0,0 +1,20 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {Hero} from "../../model/hero";
+
+@Component({
+  selector: 'mu-detail-data',
+  templateUrl: './detail-data.component.html',
+  styleUrls: ['./detail-data.component.scss']
+})
+export class DetailDataComponent implements OnInit {
+
+  constructor() {
+  }
+
+  @Input() heroData!: Hero;
```

Comment:
also since the component is related to any detail to display, can you rename the variable to be component specific and not hero specific?